src/spi.rs

```diff
@@ -23,19 +23,50 @@ pub enum Error {
     _Extensible,
 }
 
-pub trait Pins<SPI> {
-    const REMAP: bool;
-}
+// FIXME these should be "closed" traits
```

Comment:
The pin traits cannot be public, the link you provided makes the unsafe to deter implementaton but I think they should be [sealed](https://rust-lang.github.io/api-guidelines/future-proofing.html).